dotcom-rendering/src/web/components/Contributor.tsx

```diff
@@ -15,25 +16,46 @@ import TwitterIcon from '../../static/icons/twitter.svg';
 import { interactiveLegacyClasses } from '../layouts/lib/interactiveLegacyStyling';
 import { decidePalette } from '../lib/decidePalette';
 
-const twitterHandleStyles = (palette: Palette) => css`
+const twitterHandleColour = (palette: Palette) => css`
+	color: ${palette.text.twitterHandleBelowDesktop};
+
```

Comment:
Just an observation (and this is _very_ subjective), maybe the different groups of css properties can be grouped together without spaces in-between? Personally I would find it easier to distinguish them like that but hey, we all feel space differently!